pangeo_forge_runner/commands/bake.py

```diff
@@ -97,19 +123,18 @@ def start(self):
 
             for name, recipe in recipes.items():
                 # Unique name for running this particular recipe.
-                # FIXME: Should include the name of repo / ref as well somehow
-                job_name = (
-                    f"{name}-{recipe.sha256().hex()}-{int(datetime.now().timestamp())}"
-                )
+                if not self.job_name:
+                    # FIXME: Should include the name of repo / ref as well somehow
+                    self.job_name = f"{name}-{recipe.sha256().hex()}-{int(datetime.now().timestamp())}"
```

Comment:
```suggestion
                    job_name = f"{name}-{recipe.sha256().hex()}-{int(datetime.now().timestamp())}"
                else:
                    job_name = self.job_name
```

Easier to see here that `job_name` is always not None even if `self.job_name` is.